app/subscribers/super_good/solidus_taxjar/spree/reporting_subscriber.rb

```diff
@@ -19,6 +19,7 @@ def report_transaction(event)
 
           unless order.completed_at && SuperGood::SolidusTaxjar.configuration.preferred_reporting_enabled_at > order.completed_at
             order.taxjar_transaction_sync_logs.create!(status: :error, error_message: ORDER_TOO_OLD_MESSAGE)
+            SuperGood::SolidusTaxjar.exception_handler.call(RuntimeError.new(ORDER_TOO_OLD_MESSAGE))
```

Comment:
I am a bit torn on this, if it's an expected outcome that we don't report orders that are created prior to the backfill period start, is it really an exception?

Reply:
I don't feel strongly about this. A possible alternative: what would be the drawbacks of emitting an INFO level log line instead?

Reply:
Discussed IRL and the consensus here the more visibility the better.

The reason an INFO log message is not ideal is that it could possibly go nowhere, so we are leveraging the user configured exception handler instead.